pkg/operator/ceph/pool/status.go

```diff
@@ -130,3 +131,28 @@ func toCustomResourceStatus(currentStatus *cephv1.MirroringStatusSpec, mirroring
 
 	return mirroringStatusSpec, mirroringInfoSpec, snapshotScheduleStatusSpec
 }
+
+func buildStatusInfo(cephBlockPool *cephv1.CephBlockPool) map[string]string {
+	m := make(map[string]string)
+	if cephBlockPool.Spec.Mirroring.Enabled {
+		mirroringInfo := opcontroller.GenerateStatusInfo(cephBlockPool)
+		for key, value := range mirroringInfo {
+			logger.Info("In the loop now!")
```

Comment:
update this log to sepcific value

Reply:
Good catch was left from debugging, will remove